tips/tips-en.txt

```diff
@@ -89,18 +89,23 @@ You can easily move windows between workspaces by dragging them around in the Wo
 https://www.haiku-os.org/docs/userguide/en/workspaces.html
 %
 Terminal
-Press CTRL R and start to enter a command and you'll be provided with the first match from the bash history. Keep pressing CTRL R until you find the right command line and press ENTER to execute it.
+Press CTRL + R and start to enter a command and you'll be provided with the first match from the bash history. Keep pressing CTRL R until you find the right command line and press ENTER to execute it.
 https://www.haiku-os.org/docs/userguide/en/applications/terminal.html#shortcuts
 %
 GUI
-Use keyboard's PgUp/PgDown and Home/End to quickly navigate the context menu in a very crowded folder.
+Use keyboard's Page↑/ Page↓ and Home/End to quickly navigate the context menu in a very crowded folder.
 https://www.haiku-os.org/docs/userguide/en/tracker.html
 %
 Miscellaneous
-You can open a document with an already running app by dragging its icon onto the apps' enty in the Deskbar.
+You can open a document with an already running app by dragging its icon onto the apps entry in the Deskbar.
 %
 Miscellaneous
-Hold down the right ALT+SHIFT+CRTL and click a program in the Deskbar to kill it.
+Hold down the right ALT+SHIFT+CTRL and click a program in the Deskbar to kill it this is called Vulcan Death Grip .
```

Comment:
We have to stay consistent: space before and after the "+".
Full stop after "...kill it." then "This is the Vulcan Death Grip." (No space in front of ".")